stl/inc/xnode_handle.h

```diff
@@ -80,21 +80,21 @@ class _Node_handle : public _Base<_Node_handle<_Node, _Alloc, _Base, _Types...>,
     aligned_union_t<0, _Alloc> _Alloc_storage; // Invariant: contains a live _Alloc iff _Ptr != nullptr
 
     void _Clear() noexcept { // destroy any contained node and return to the empty state
-        if (_Ptr) {
+        if (_Ptr != nullptr) {
             _Alloc& _Al = _Getal();
             _Alnode _Node_alloc{_Al};
-            _Alnode_traits::destroy(_Node_alloc, _STD addressof(_Ptr->_Myval));
+            _Alty_traits::destroy(_Al, _STD addressof(_Ptr->_Myval));

```

Comment:
Should reorder this before the rebind above

Reply:
Done, both places.